assert/equal.go

```diff
@@ -31,3 +72,7 @@ func errordetails(details ...interface{}) string {
 	}
 	return ""
 }
+
+func floatEqual(a, b float64) bool {
+	return math.Abs(a-b) < ε && math.Abs(b-a) < ε
```

Comment:
A little curious on why we need to use math.Abs for a-b and b-a (some floating point corner case ?). Also I usually allowed the client to define the desired precision when building this sort of equal assert function, but one with a default value also seems great =D

Reply:
Sorry. Probably this doesn't makes sense hehe
Well, what I was attempting to do here was test if a is close to b from both sides. Think in the Cartesian plan when a > 0 and b < 0. But... math.Abs already solves this.
I just checked how I had made this in the past and yes, I have just made `math.Abs(a-b) < epsilon`:
https://github.com/NeowayLabs/signal/blob/master/signal.go#L16

About allowing the user to set epsilon, I think makes sense also but a default floatEqual implementations that just works for most cases should be good. I hate having to think about this every time I work with floats.

I'll fix soon.